content/blog/reliable-local-development-for-infrequently-used-projects.md

```diff
@@ -0,0 +1,143 @@
+---
+id: blog
+title: Reliable codified development setups with GitPod
+description: Local development setups are hard, and they often don't age well. This blog
+  outlines how you can fix that with codified cloud-based development setups using GitPod
+author: Benny Cornelissen
+tags:
+  - cloud
+  - coding
+  - productivity
+slug: reliable-codified-development-setups-gitpod
+date: 2022-09-19
+images:
+  - "/img/blog/gitpod-illustration.png"
+---
+
+Local development is hard. Yes, if you pour enough time into it, you can have local setups that work fine, some might even be elegant. But not everyone has that kind of time, and not every project warrants it. Too often I find that in order to contribute to an open source project, I first have to set up all kinds of stuff to even get started. And it's one thing if it's a project that I use a lot, and have intimate knowledge of, and is built on a tech stack I'm good at. Because that'll probably mean that it's fairly straightforward to get the development setup working properly, and it will also be worth the time I put into it.
```

Comment:
Personally, I would sprinkle in some whitespace here to let the text "breathe" a little bit more.

"Local development is hard. 

Yes, if you pour enough time into it, you can have local setups that work fine, some might even be elegant. But not everyone has that kind of time, and not every project warrants it. 

Too often I find that in order to contribute to an open source project, I first have to set up all kinds of stuff to even get started. And it's one thing if it's a project that I use a lot, and have intimate knowledge of, and is built on a tech stack I'm good at. Because that'll probably mean that it's fairly straightforward to get the development setup working properly, and it will also be worth the time I put into it."

Reply:
This is personal opinion of course, but to me it makes the beginning of the article a little bit more readable, inviting people to read on.

Reply:
Just tried it. Looks better to me, so I'm adopting your suggestion